morphic.js

```diff
@@ -7682,19 +7682,25 @@ MenuMorph.prototype.unselectAllItems = function () {
     this.changed();
 };
 
-MenuMorph.prototype.popup = function (world, pos) {
+MenuMorph.prototype.popup = function (world, pos, submenu) {
```

Comment:
It's not possible without some refactoring apparently (see 2 comments ago from BH), but he said he'll look into it. :)